healthz/handlers/healthz.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	"net/http"
+	"io"
+	"github.com/gravitational/satellite/healthz/clusterstatus"
+	"github.com/gravitational/satellite/healthz/config"
+)
+
+func HandlerHealthz(w http.ResponseWriter, req *http.Request) {
+	config := req.Context().Value(config.ContextKey).(config.Config)
+	clusterStatus := req.Context().Value(clusterstatus.ContextKey).(clusterstatus.ClusterStatus)
+
+	if req.FormValue("accesskey") != config.AccessKey {
+		http.Error(w, "401 Unauthorized", http.StatusUnauthorized)
```

Comment:
Redundant code in the message - the browser or client can get the code from the header